packages/context/src/context-subscription.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright IBM Corp. 2017,2018. All Rights Reserved.
+// Node module: @loopback/context
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {ContextEventObserver, Subscription} from './context-observer';
+
+/**
+ * An implementation of `Subscription` interface for context events
+ */
+export class ContextSubscription implements Subscription {
+  constructor(
+    protected notifier: {unsubscribe(observer: ContextEventObserver): boolean},
```

Comment:
Why not `ContextEventNotifier`? It might be overkill to have anonymous interface here for `unsubscribe`.